### PR TITLE
Build version 1.0.5:2

### PR DESCRIPTION
**Pull request by Andrewsopoku**
**Issue**

- Link:
- Summary: Build version increased

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
Doesn't package.json need to be bumped too?

**Comment by marvinmarnold** [comment]
@Andrewsopoku ^^

**Comment by Andrewsopoku** [comment]
@Andrewsopoku added. Forgot to save.

File: package.json [hunk 1]
```diff
@@ -1,167 +1,167 @@
...
-    "clean-start": "yarn start -- --reset-cache",
-    "postinstall": "yarn patch-package",
-    "bump-patch": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type patch",
-    "bump-minor": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type minor",
-    "bump-major": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type major"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "Nebra",
```
Comment (marvinmarnold) [inline package.json:2]: It's distracting when the indentation changes. Hard to make sure only the expected changes are included.

This is a JS package, so I'm inclined to say 2 spaces is more correct.

**Review by marvinmarnold — COMMENTED** [review]
Just want to address indenting.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Nebra",
-  "version": "1.0.4",
```
Comment (marvinmarnold) [inline package.json:3]: You accidentally omitted the `5`

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
Still a typo

**Comment by Andrewsopoku** [comment]
@marvinmarnold Deploying to Testflight but encountered this issue.
sds

**Comment by shawaj** [comment]
@marvinmarnold done

**Comment by marvinmarnold** [comment]
Thanks @shawaj. @Andrewsopoku can you redeploy the newest code for iOS? I'll take care of Android later.

**Review by marvinmarnold — APPROVED** [review]

**Comment by marvinmarnold** [comment]
@Andrewsopoku in future release PRs, can you add bullet points for the changelog?

**Pull request merged**